pkgs/misc/emulators/yuzu-mainline/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchgit, cmake, libressl, wrapQtAppsHook, SDL2, qtbase, python2 }:
+
+stdenv.mkDerivation {
+  pname = "yuzu-mainline";
+  version = "unstable-2020-04-13";
+
+  src = fetchgit {
+    url = "https://github.com/yuzu-emu/yuzu-mainline";
+    rev = "20c42dc9ea8bdc208d9d274350af2ff347876c76";
+    sha256 = "0s6vnvbyshbrl1xxiwdrxc2jl75jyjlf4kidl88c7pwj3m2gcwss";
+    leaveDotGit = true;
+    fetchSubmodules = true;
+  };
+
+  enableParallelBuilding = true;
+  nativeBuildInputs = [ cmake libressl wrapQtAppsHook ];
+  cmakeFlags = "-DCMAKE_BUILD_TYPE=Release";
+  buildInputs = [ SDL2 qtbase python2 ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://yuzu-emu.org";
+    description = "An experimental open-source emulator for the Nintendo Switch";
```

Comment:
I know you're using the official wording from the readme, but this description is pretty confusing. This is a Nintendo Switch emulator, not an emulator for the Switch.

How about this:
 ` description = "An experimental Nintendo Switch emulator";`